app/src/main/java/org/p2p/wallet/settings/DeviceInfoHelper.kt

```diff
@@ -0,0 +1,10 @@
+package org.p2p.wallet.settings
+
+import android.os.Build
+
+object DeviceInfoHelper {
+
+    fun getCurrentDeviceName(): String {
+        return Build.MANUFACTURER + ' ' + Build.MODEL
```

Comment:
I suggest replacing all the occurrences.

Reply:
kek they already replaced =)